src/frontend/komponenter/ManuellJournalføring/ManuellJournalføring.tsx

```diff
@@ -19,8 +19,21 @@ import HentPerson from '../Felleskomponenter/HentPerson/HentPerson';
 import UIModalWrapper from '../Felleskomponenter/Modal/UIModalWrapper';
 import Skjemasteg from '../Felleskomponenter/Skjemasteg/Skjemasteg';
 import { KnyttTilBehandling } from './KnyttTilBehandling';
+import { Dokumenter } from './Dokumenter';
+import styled from 'styled-components';
 import { useAmplitude } from '../../utils/amplitude';
 
+const PageSplit = styled.div`
```

Comment:
```suggestion
const Sidesplit = styled.div`
```
Is this the actual split or just flexin? I think it would be nice to gather the whole pagesplit logic. Maybe something like:
<Sideplit>
<venstreSide>
<høyreSide>
<sidesplit/>

Reply:
I agree that it would be nice to rename this to norwegian :) when I see an english named component without context I automatically assume that it is an external component